src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceImplTest {
+    @BeforeAll
+    static void beforeAll() {
+        WriterService writerService = new WriterServiceImpl();
+        String filePath = "src/test/resources/data3.txt";
+        StringBuilder report = new StringBuilder().append("fruit,quantity")
+                .append(System.lineSeparator()).append("banana,100")
+                .append(System.lineSeparator()).append("apple,15");
+        writerService.createReportFile(report.toString(), filePath);
+    }
+
+    @Test
+    void createReportFile_Ok() {
+        try {
+            final List<String> actual = Files.readAllLines(Path.of("src/test/resources/data3.txt"));
+            List<String> expected = new ArrayList<>();
+            expected.add("fruit,quantity");
+            expected.add("banana,100");
+            expected.add("apple,15");
+            assertEquals(expected, actual);
+        } catch (IOException e) {
+            fail();
+        }
+    }

```

Comment:
You don't test your class' methods here

Reply:
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/126912806/181ace99-f101-457a-9f5c-4d6b76a94257)
I disagree because if I change the data the test will fail